Raven.Voron/Voron/Impl/Backup/IncrementalBackup.cs

```diff
@@ -143,10 +143,10 @@ public string JournalLocation
                                 header->IncrementalBackup.LastBackedUpJournalPage = lastBackedUpPage;
                             });
                         }
-                        catch (Exception)
+                        catch (Exception e)
                         {
                             backupSuccess = false;
-                            throw;
+                            throw e;
```

Comment:
Why are you doing this and killing the stack trace here?

Reply:
Debugging artifact I forgot to remove. Will fix this.